src/Bitmap_cubical_complex/include/gudhi/Bitmap_cubical_complex_base.h

```diff
@@ -109,6 +109,15 @@ class Bitmap_cubical_complex_base {
    **/
   virtual inline std::vector<std::size_t> get_coboundary_of_a_cell(std::size_t cell) const;
 
+  /**
+   * This function finds a top-dimensional cell that is incident to the input cell and has 
+   * the same filtration value. In case several cells are suitable, an arbitrary one is 
+   * returned. Note that the input parameter is not necessarily a cube, it might also
+   * be an edge or vertex of a cube. On the other hand, the output is always indicating the position of
+   * a cube in the data structure. 
+   **/
+  inline int get_top_dimensional_coface_of_a_cell(int splx);
```

Comment:
int -> size_t (check for other places where this may be needed)

Reply:
Thanks. Done.

Reply:
There were two `int` on that line but you only changed one?